### PR TITLE
[ShapeOpt] fix gid default and bead optimization test

### PR DESCRIPTION
**Pull request by armingeiser**
Minor fixes

### CODE REVIEW

**Comment by philbucher** [comment]
Can you also change the solver to a non-intel one?
Otherwise it fails in the nightly build

File: applications/ShapeOptimizationApplication/tests/algorithm_bead_optimization_test/analysis_parameters.json [hunk 1]
```diff
@@ -30,9 +30,7 @@
       "residual_absolute_tolerance"        : 1e-9,
       "max_iteration"                      : 10,
       "linear_solver_settings"             : {
-          "solver_type" : "eigen_pardiso_lu",
-          "scaling"     : false,
-          "verbosity"   : 0
+          "solver_type" : "eigen_sparse_lu"
```
Comment (philbucher) [inline applications/ShapeOptimizationApplication/tests/algorithm_bead_optimization_test/analysis_parameters.json]: :+1: 
Does it have a skiptest if EigenSolversApp is not available?

=> I think @dbaumgaertner told me that for this case also the SuperLU would work, but you better check with him

Comment (armingeiser) [inline applications/ShapeOptimizationApplication/tests/algorithm_bead_optimization_test/analysis_parameters.json]:
```suggestion
          "solver_type" : "SuperLUSolver"
```

**Comment by dbaumgaertner** [comment]
Ah, I prepared a PR for this, but obviously you guys were faster. Thanks a lot. I just removed another intel dependency. SuperLU is slower but fine.

**Review by dbaumgaertner — APPROVED** [review]

**Pull request merged**